src/store/main.ts

```diff
@@ -1877,58 +1877,19 @@ export class MainStore {
     this.activeOrg = org;
   }
 
-  async getOrganizationNextBountyByCreated(org_uuid: string, bountyId: string): Promise<number> {
+  async getOrganizationNextBountyByCreated(org_uuid: string, created: number): Promise<number> {
     try {
-      const params = { languages: this.bountyLanguages, ...this.orgBountiesStatus };
-
-      const queryParams: QueryParams = {
-        limit: queryLimit,
-        sortBy: 'created',
-        search: uiStore.searchText ?? '',
-        page: 1,
-        resetPage: false,
-        ...params
-      };
-
-      // if we don't pass the params, we should use previous params for invalidate query
-      const query = this.appendQueryParams(
-        `gobounties/org/next/${org_uuid}/${bountyId}`,
-        queryLimit,
-        queryParams
-      );
-
-      const bounty = await api.get(query);
+      const bounty = await api.get(`gobounties/org/next/${org_uuid}/${created}`);
       return bounty;
     } catch (e) {
       console.log('fetch failed getOrganizationNextBountyById: ', e);
       return 0;
     }
   }
 
-  async getOrganizationPreviousBountyByCreated(
-    org_uuid: string,
-    bountyId: string
-  ): Promise<number> {
+  async getOrganizationPreviousBountyByCreated(org_uuid: string, created: number): Promise<number> {
     try {
-      const params = { languages: this.bountyLanguages, ...this.orgBountiesStatus };
-
-      const queryParams: QueryParams = {

```

Comment:
This won't fix it @Vayras, the issue is when the left and right arrow is clicked the Status goes back to the default state and all the statuses become false again, hence the wrong data.